README.md

```diff
@@ -25,6 +25,23 @@ cd path/to/sublime-text/Packages
 git clone https://github.com/tinted-theming/sublime-text tinted_theming
 ```
 
+### [Package Control](https://packagecontrol.io/) users
+
+1. Open command palette > Preferences: Package Control Settings
+2. Add the following to `Package Control.sublime-settings` and save:
+
+```json
+"repositories": [
+	"https://github.com/tinted-theming/tinted-sublime-text"
```

Comment:
```suggestion
    "https://github.com/tinted-theming/tinted-sublime-text"
```

Reply:
@JamyGolden maybe it's simpler to accept pr and then fix the spacing by yourself? it's been almost 3 weeks and @alexwforsythe does not react, but this is a useful pr for users

Reply:
Fair enough. I just commented and forgot about the PR. I’ll merge now.